src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {

```

Comment:
```suggestion
public class Lottery {
```
the same as Application. Lottery should not have properties like color or number,